tests/run-centos-ci.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# setting up virtual environment
+yum -y install epel-release
+virtualenv --system-site-packaes env
+source env
```

Comment:
You probably want `source env/bin/activate`

Reply:
Done